recipes/GIGATL/meta.yaml

```diff
@@ -0,0 +1,24 @@
+title: "SWOT-AdAC data (GIGATL)"
+description: "GIGATL regional data for SWOT-AdAC Xovers"
+pangeo_forge_version: "0.3.3"
+pangeo_notebook_version: "2021.05.04"
+recipes:
+  - dict_object: "recipe:recipes"
+provenance:
+  providers:
+    - name: "LOPS"
+      description: "Laboratoire d'Oceaographie Physique et Spatiale"
+      roles:
+        - producer
+        - licensor
+      url: https://github.com/roxyboy/SWOT-AdAC-ocean-model-intercomparison
+  license: "CC-BY-4.0"
+maintainers:
+  - name: "Charles Stern"
+    github: cisaacstern
```

Comment:
Need to add @roxyboy here!

Reply:
@sharkinsspatial is this the correct way to provide multiple maintainers in meta.yaml:

https://github.com/pangeo-forge/staged-recipes/pull/27/files#diff-d38a3209f5f32e0a94f8ae175d65eb93b198b656fbb117e61742eaa329ad3795R16-R20

?